backend/app/routes/decisions.py

```
@@ -0,0 +1,422 @@
+                outcome=decision.outcome if decision else None,
+                outcome_summary=decision.outcome_summary if decision else None,
+                duration_seconds=duration,
+                human_review_status=task.human_review_status,
+                created_at=_iso(task.created_at),
+            )
+        )
+    return {"decisions": summaries, "total": len(summaries)}
+
+
+@router.get("/chain/verify")
+def verify_full_chain(db: Session = Depends(get_db)):
```

Ruff (0.16.2)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)